app/views/directives/bind-service/delete-binding-result.html

```diff
@@ -5,13 +5,12 @@ <h3 class="mar-top-none">
     </h3>
 
     <div
-      ng-if="ctrl.appsForBinding(ctrl.selectedBinding)"
-      ng-repeat="appForBinding in ctrl.appsForBinding(ctrl.selectedBinding)">
+      ng-if="ctrl.appsForBinding(ctrl.selectedBinding.metadata.name)"
+      ng-repeat="appForBinding in ctrl.appsForBinding(ctrl.selectedBinding.metadata.name)">
       {{appForBinding.metadata.name}} <small class="text-muted">&ndash; {{ appForBinding.kind | humanizeKind : true}}</small>
     </div>
-    <div ng-if="!(ctrl.appsForBinding(ctrl.selectedBinding))">
-      <!-- TODO: selected a secret name in the form, but see a binding name in the result. -->
-      {{ctrl.selectedBinding}} <small class="text-muted">&ndash; Binding</small>
+    <div ng-if="!(ctrl.appsForBinding(ctrl.selectedBinding.metadata.name))">
```

Comment:
Probably should use the `size` filter. In case we ever change the overview to set an empty array instead of null/undefined.